introduction.md

```diff
@@ -0,0 +1 @@
+Hola, mi nombre es Isaac Yriarte y estoy haciendo el curso de Code School Mastering Github
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.